Makefile

```diff
@@ -129,6 +129,11 @@ cli-clean:
 
 # Install CLI
 cli-install:
+	@echo "Please installed a released version of KUDO."
```

Comment:
If you have source code checked out then you're probably not a typical _end-user_, right? I'm not sure what the benefit of this rename should be.

Reply:
> If you have source code checked out then you're probably not a typical _end-user_, right?

One would think so, right? Well, it turns out it's not true - #1549 is a living proof.